### ISSUE TITLE
Do not query the schedd and always use the node_status file

### ISSUE THREAD

**Issue by mmascher** [dmwm/CRABServer#4617]
From Brian's email here  https://hypernews.cern.ch/HyperNews/CMS/get/crabDevelopment/2252.html

4) (HTCondor bug #4607) DAGMan can be told to always update the node_status file.  Hence, if the last update time in the node_status file is up-to-date within the minute, we have no need to query the schedd to do "crab status".  This is pretty significant - it means that, if we download node_status first, we no longer need to query the schedd for the wide majority of "crab status" invocations.
- Basically, with regards to this particular "feature", CRAB3 can now catch up with CRAB2.
- This requires some developer time to utilize & validate.
- #4686 fixed a few more cases where the node_status file may not get updated on "crab kill"

**Comment by belforte** [comment]
wasn't this done ? why still open ?

**Comment by mmascher** [comment]
Hi, we are still doing this query: https://github.com/dmwm/CRABServer/blob/master/src/python/CRABInterface/HTCondorDataWorkflow.py#L47-L59

The basic `crab status` invocation uses this to get the status of the dag and the DagmanHoldReason in case of failure if I am not wrong.
